src/main/java/com.gigssandbox/UserMessenger.java

```diff
@@ -0,0 +1,79 @@
+package com.gigssandbox;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Properties;
+import java.util.Scanner;
+
+class UserMessenger {
+
+    private static final Scanner in = new Scanner(System.in);
+    private Properties properties = new Properties();
```

Comment:
You've better init class fields only inside constructor

Reply:
![1](https://user-images.githubusercontent.com/12383964/50538668-c679c080-0b83-11e9-9862-1e5632f69ffc.jpg)
how to deal with this issue?

Reply:
what is the purpose making this Scanner static?

Reply:
I don't remember why I have made this static, now it's really not necessary